server/metrics/search.go

```diff
@@ -33,6 +33,7 @@ func getSearchOkTagKeys() []string {
 		"env",
 		"project",
 		"db",
+		"branch",
```

Comment:
Should this be here?

Reply:
In this context, this tag is not helpful. This is used by the low-level storage, there we don't need branch-specific tags.